class-vip-support-user.php

```diff
@@ -473,6 +475,12 @@ public function action_parse_request() {
 			wp_die( $rebuffal_message, $rebuffal_title, array( 'response' => 403 ) );
 		}
 
+		if ( ! is_proxied_automattician() ) {
```

Comment:
Same concern here - this requires a verified email to run. We probably want to check the constant directly - `A8C_PROXIED_REQUEST`, rather than the `is_automattician()` function.